infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -105,6 +113,43 @@
     @Getter
     private volatile CompletableFuture<Boolean> sequencerBootstrappedFuture;
 
+    /**
+     * Timeout before which the management agent attempts to bootstrap a NOT_READY sequencer.
+     */
+    private static final int sequencerStateHeartBeatCounter = 3;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)